test/lib/jdk/test/lib/process/ProcessTools.java

```diff
@@ -894,7 +896,7 @@ public static void main(String[] args) throws Throwable {
             // MainThreadGroup used just as a container for exceptions
             // when main is executed in virtual thread
             MainThreadGroup tg = new MainThreadGroup();
-            Thread vthread = startVirtualThread(() -> {
+            Thread vthread = createVirtualThread(() -> {
```

Comment:
You can replace this with `Thread.ofVirtual().unstarted(() -> {` and remove the helper method. I think that helper method dates from the jtreg wrapper solution.

Reply:
Yes, I already brought this up with Leonid. I was going to file a separate bug for it, but I can fix it with this CR if you'd like.